app/components/case-studies-components/CaseStudyProblemStatement.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import Image from "next/image";
+
+import { CaseStudyHeaderProps } from "@/utils/interfaces";
+import { workProcessSymbols } from "@/constants";
+
+const CaseStudyProblemStatement: React.FC<CaseStudyHeaderProps> = ({
+  mainProject,
+}) => {
+  return (
+    <>
+      <section className="flex w-full flex-col items-center justify-center bg-white800 px-6 py-9 dark:bg-black300 md:py-[4.5rem]">
+        <div className="flex flex-col gap-4 self-center md:max-w-3xl">
+          {mainProject?.caseStudyDescription?.map((text, index) => (
+            <p
+              key={index}
+              className="text-sm font-light text-white500 dark:text-white800 md:text-xl"
+            >
+              {text}
+            </p>
+          ))}
+        </div>
+      </section>
+      <section className="flex w-full flex-col items-center justify-center bg-white900 px-6 py-9 dark:bg-black200 md:py-[4.5rem]">
+        <div className="flex w-full flex-col md:max-w-3xl">
+          <p className="text-xs font-semibold text-primaryLight dark:text-primaryDark md:text-sm">
+            Problem
+          </p>
+          <p className="mt-2 text-[1.75rem] font-semibold text-black200 dark:text-white900 md:text-[2rem]">
+            Problem Statement
+          </p>
+          <p className="mt-5 text-sm font-light text-white500 dark:text-white800 md:text-xl">
+            {mainProject?.problemStatement}
+          </p>
+          {mainProject?.problemStatementImage && (
+            <div className="mt-6 w-full max-w-3xl md:mt-8">
+              <Image
+                src={mainProject.problemStatementImage}
+                alt="image underlying text in problem statement"
+                className="w-full rounded md:rounded-xl"
+              />
+            </div>
+          )}
+        </div>
+      </section>
+      {mainProject?.figmaBannerImage && (
+        <section className="flex w-full flex-col items-center justify-center">
+          <div className="flex w-full items-center justify-center bg-primaryLight py-2.5 text-sm font-semibold text-white900 dark:bg-primaryDark md:py-9 md:text-[2rem]">
+            HIGH-FIDELITY FIGMA DESIGN
+          </div>
+          <div className="w-full">
+            <Image
```

Comment:
Same here. It needs width and height unless you're using fill.

Reply:
@cdsalty I have now put layout='responsive' here instead of fill. Is that ok